README.md

```diff
@@ -164,6 +164,10 @@ It also supports the `// +kubebuilder:validation:Optional` and `// +kubebuilder:
 
 If you prefer to use the Kubebuilder markers instead, you can change the preference in the configuration.
 
+## Nophase
```

Comment:
You've inserted this in the middle of a different linters block. Should move above the `## OptionalOrRequired` heading